content/docs/configuration/acme/README.md

```diff
@@ -383,4 +343,28 @@ solvers:
 ```
 In this case the `DNS01` solver for CloudFlare will only be used to solve a
 challenge for a DNS name if the `Certificate` has a label from
-`matchLabels` _and_ the DNS name matches a zone from `dnsZones`.
\ No newline at end of file
+`matchLabels` _and_ the DNS name matches a zone from `dnsZones`.
+
+
+## Alternative Certificate Chains
```

Comment:
Suggestion: it would be good to keep the bit of information that if the 'preferredChain' is not found then the default certificate will be returned as I imagine that this is not obvious.

Reply:
Yeah good point! I've updated now in a new commit (with some better language overall I think)